openedx/core/djangoapps/user_authn/templates/user_authn/edx_ace/passwordresetsuccess/email/body.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% filter force_escape %}
+                {% blocktrans %}Your {{ platform_name }} password has changed{% endblocktrans %}
+                {% endfilter %}
+            </h1>
```

Comment:
This block should be moved after the hello block

Reply:
@awaisdar001 that's the heading in the email template. 
I am going to change this to "Password Reset Success" 

<img width="559" alt="Screen Shot 2020-09-29 at 3 37 09 PM" src="https://user-images.githubusercontent.com/4252738/94548095-a51e9a80-0269-11eb-84b2-c3eef3b7ef55.png">